en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -94,184 +53,140 @@ Server).
+    ``` 
+    keytool -export -keystore $WEB_APP_KEYSTORE -alias wso2carbon -file webserver.cer
     ```
 
-    ``` xml
-        keytool -import -alias <alias> -keystore  $IS_HOME/repository/resources/security/client-trustore.jks -file webserver.cer
+    ``` 
+    keytool -import -alias <alias> -keystore  $IS_HOME/repository/resources/security/client-trustore.jks -file webserver.cer
     ```
 
 ## Moving the accountrecoveryendpoint from WSO2IS and hosting it on a separate web server
 
-This is an additional improvement which enables
-hosting accountrecoveryendpoint.war also on a separate web server.
+This is an additional improvement which enables hosting `accountrecoveryendpoint.war` also on a separate web server.
 
 !!! tip "Before you begin"
-    Get a copy of
-    `         <IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint.war        `
-    to `         <WebApp_HOME>/        ` and unzip it. Make sure to get the
-    `         accountrecoveryendpoint.war        ` after the WUM update and
-    not the extracted `         accountrecoveryendpoint        ` in
-    `         <IS_HOME>/repository/deployment/server/webapps/        `
+    Get a copy of `<IS_HOME>/repository/deployment/server/webapps/accountrecoveryendpoint.war` to `<WEBAPP_HOME>/` and unzip it. Make sure to get the `accountrecoveryendpoint.war` after the WUM update and not the extracted `accountrecoveryendpoint` in `<IS_HOME>/repository/deployment/server/webapps`.
     
 
-1.  In
-    `           <WebApp_HOME>/accountrecoveryendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties          `
-    file, uncomment and change it to identity server.
+1.  Open the `<WEBAPP_HOME>/accountrecoveryendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties` file, uncomment the following config, and change it to `identity server`.
 
     ``` xml
     identity.server.service.contextURL=https://localhost:9443/services/
     ```
 
-2.  Uncomment and change the user portal reference in
-    `           <WebApp_HOME>/accountrecoveryendpoint/WEB-INF/web.xml          `
+2.  Uncomment and change the user portal reference in `<WEBAPP_HOME>/accountrecoveryendpoint/WEB-INF/web.xml`.
 
     ``` xml
-        <context-param>
-                <param-name>UserPortalUrl</param-name>
-                <param-value>https://localhost:9443/dashboard/index.jag</param-value>
-        </context-param>
+    <context-param>
+            <param-name>UserPortalUrl</param-name>
+            <param-value>https://localhost:9443/dashboard/index.jag</param-value>
+    </context-param>
     ```
 
 3.  Export the following paths.
 
-    ``` xml
-        export WEB_APP_HOME=/Users/userfoo/apache-tomcat-7.0.81/webapps
-        export IS_HOME=/Users/userfoo/wso2is-5.6.0
-        export WEB_APP_LIB=${WEB_APP_HOME}/accountrecoveryendpoint/WEB-INF/lib/
+    ``` 
+    export WEB_APP_HOME=/Users/userfoo/apache-tomcat-7.0.81/webapps
```

Comment:
`apache-tomcat-7.0.81` >> change to tomcat-9

Reply:
`/Users/userfoo/apache-tomcat-7.0.81/` > `TOMCAT_HOME` and define the tomcat home at the beginning where tomcat server is observed.